htdocs/compta/paymentbybanktransfer/index.php

```diff
@@ -91,17 +95,23 @@
 print '</a>';
 print '</td></tr>';
 
+print '<tr class="oddeven"><td>'.$langs->trans("NbOfInvoiceToPayByBankTransferForSalaries").'</td>';
+print '<td class="right">';
+print '<a class="badge badge-info" href="'.DOL_URL_ROOT.'/compta/prelevement/demandes.php?status=0&type=bank-transfer&sourcetype=salary">';
+print $bprev->nbOfInvoiceToPay('bank-transfer', 'salary');
+print '</a>';
+print '</td></tr>';
+
 print '<tr class="oddeven"><td>'.$langs->trans("AmountToTransfer").'</td>';
 print '<td class="right"><span class="amount nowraponall">';
-print price($bprev->SommeAPrelever('bank-transfer'), '', '', 1, -1, -1, 'auto');
+print price(($bprev->SommeAPrelever('bank-transfer') + $bprev->SommeAPrelever('bank-transfer', 'salary')), '', '', 1, -1, -1, 'auto');
 print '</span></td></tr></table></div><br>';
 
 
-
 /*
  * Invoices waiting for withdraw
  */
-$sql = "SELECT f.ref, f.rowid, f.total_ttc, f.fk_statut, f.paye, f.type, f.datef, f.date_lim_reglement,";
+$sql = "SELECT f.ref, f.rowid, f.salary, f.paye,";$sql = "SELECT f.ref, f.rowid, f.total_ttc, f.fk_statut, f.paye, f.type, f.datef, f.date_lim_reglement,";
```

Comment:
This line of code seems bugged.
`$sql = "SELECT f.ref, f.rowid, f.salary, f.paye,";$sql = "SELECT f.ref, f.rowid, f.total_ttc, f.fk_statut, f.paye, f.type, f.datef, f.date_lim_reglement,";`
shoudl be 
`$sql = "SELECT f.ref, f.rowid, f.total_ttc, f.fk_statut, f.paye, f.type, f.datef, f.date_lim_reglement,";`

Reply:
yes, I didn't pay attention